### DISCUSSION TITLE
Cant Load

### DISCUSSION THREAD

**Discussion by ferikurniawan86** [jgchristopher/obsidian-clipper#42] [General]
![image](https://github.com/jgchristopher/obsidian-clipper/assets/129614096/382f2dc6-3b5e-4de9-b87c-8bb87a3361ba)

**Comment by jgchristopher** [comment]
I will test this to see if I can reproduce the issue.

**Comment by joachimVdH** [comment]
Hi, I think I ran in the same kind of issue.
On Edge (115.0.1901.203) :
<img width="391" alt="screenshot 2023-08-19 om 15 22 42" src="https://github.com/jgchristopher/obsidian-clipper/assets/33933/6db471b2-12c7-4ce2-b71f-def95bb7fec5">

When I added following to the manifest.json
```
,
	"manifest_version": 3
```

it loaded successfully

**Comment by jgchristopher** [comment]
oh, is this for the clipper chrome extension?

**Reply by joachimVdH** [reply]
indeed, obsidian-clipper ( https://github.com/jgchristopher/obsidian-clipper/releases/download/0.2.1/obsidian-clipper-0.2.1.zip )

**Comment by jgchristopher** [comment]
Ok, I just pushed out a new release. I noticed that the  `.zip` file built didn't include the `styles.css` file, so I fixed that. 

The manifest.json file is the same one that Obsidian uses to install the plugin via the UI, so I don't know why that isn't working if you download the zip file and try to install it that way.
